src/modules/item/utils.ts

```diff
@@ -461,6 +458,51 @@ export function areSynced(item: Item, entity: Entity) {
   return true
 }
 
+export function isEmoteSynced(item: Item, entity: Entity) {
+  if (item.type !== ItemType.EMOTE) {
+    throw new Error('Item must be EMOTE')
+  }
+
+  // check if metadata has the new schema from ADR 74
+  const isADR74 = 'emoteDataADR74' in entity.metadata
```

Comment:
This is here because there are already some emotes deployed that don;t have the new schema from ADR 74. We will mark those as unsynced, so the owner (which is only Shibu so far) has to do a Push Chages + Approve, this will upload the new entities to the catalyst with the rht type and schema